.circleci/config.yml

```diff
@@ -105,6 +105,7 @@ workflows:
             branches:
               only: master
```

Comment:
i think youll need the context here as well

Reply:
Good call - added